com.tencent.WeChat.desktop

```
@@ -8,4 +8,5 @@ Icon=com.tencent.WeChat
 StartupWMClass=WeChat
 Categories=Network;
 Comment=WeChat Desktop
+Keywords=wechat,weixin
```

```suggestion
Keywords=wechat;weixin;
```

Be consistent with line 9